### ISSUE TITLE
Warnings truncated on the result web view

### ISSUE THREAD

**Issue by lemmy** [alygin/vscode-tlaplus#179]
TLC command-line:

```
*** Warnings: 2
line 63, col 1 to line 67, col 31 of module SequencesExt
Warning: the definition of 'Contains' conflicts with 
its definition at line 19, col 1 to line 23, col 31 of module Utils.
line 116, col 1 to line 120, col 45 of module SequencesExt
Warning: the definition of 'RemoveAt' conflicts with 
its definition at line 25, col 1 to line 29, col 45 of module Utils.
```

![Screenshot from 2020-12-04 13-19-49](https://user-images.githubusercontent.com/88777/101215886-6aebc380-3633-11eb-9e0b-08c3b70ceb66.png)

**Comment by alygin** [comment]
Fixed in the [v1.5.2-beta.1 pre-release](https://github.com/alygin/vscode-tlaplus/releases/tag/v1.5.2-beta.1).

**Comment by alygin** [comment]
Fix published in v1.5.2
